docs/D-Modularity/pointers.md

```diff
@@ -309,143 +321,143 @@ a is 6
-    <td align="center" bgcolor="#FFFF99">6</td> </tr>
-
-<tr><td align="center" bgcolor="#FFFF99">0x0012FF88</td>
-    <td align="center" bgcolor="#FFFF99">0x0012FF84</td>
-    <td align="center" bgcolor="#FFFF99">?</td>
-    <td align="center">5</td>
-    <td align="center">6</td> </tr>
-
-<tr><td align="center" bgcolor="#FFFF99">0x0012FF88</td>
-    <td align="center" bgcolor="#FFFF99">0x0012FF84</td>
-    <td align="center" bgcolor="#FFFF99">5</td>
-    <td align="center">5</td>
-    <td align="center">6</td> </tr>
-
-<tr><td align="center" bgcolor="#FFFF99">0x0012FF88</td>
-    <td align="center" bgcolor="#FFFF99">0x0012FF84</td>
-    <td align="center" bgcolor="#FFFF99">5</td>
-    <td align="center">5</td>
-    <td align="center">6</td> </tr>
-
-<tr><td align="center" bgcolor="#FFFF99">0x0012FF88</td>
-    <td align="center" bgcolor="#FFFF99">0x0012FF84</td>
-    <td align="center" bgcolor="#FFFF99">5</td>
-    <td align="center">5</td>
-    <td align="center">6</td> </tr>
-
-<tr><td align="center">0x0012FF88</td>
-    <td align="center">0x0012FF84</td>
-    <td align="center">5</td>
-    <td align="center" bgcolor="#FFFF99">5</td>
-    <td align="center" bgcolor="#FFFF99">6</td> </tr>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>6</code></td> </tr>
+
+<tr><td align="center"><code>0x0012FF88</code></td>
+    <td align="center"><code>0x0012FF84</code></td>
+    <td align="center">?</td>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>6</code></td> </tr>
+
+<tr><td align="center"><code>0x0012FF88</code></td>
+    <td align="center"><code>0x0012FF84</code></td>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>6</code></td> </tr>
+
+<tr><td align="center"><code>0x0012FF88</code></td>
+    <td align="center"><code>0x0012FF84</code></td>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>6</code></td>
+    <td align="center"><code>6</code></td> </tr>
+
+<tr><td align="center"><code>0x0012FF88</code></td>
+    <td align="center"><code>0x0012FF84</code></td>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>6</code></td>
+    <td align="center"><code>5</code></td> </tr>
+
+<tr><td align="center"><code>0x0012FF88</code></td>
+    <td align="center"><code>0x0012FF84</code></td>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>6</code></td>
+    <td align="center"><code>5</code></td> </tr>
 </table>
 
 Some programmers prefer symbolic notation instead of address values.  For example, they use the symbol `main::a` to refer to the local variable `a` in the function `main()`.  A walkthrough table using symbolic notation looks something like this:
 
 <table border="0">
-<tr><td align="center" colspan="3">void</td>
-    <td align="center" colspan="2">int</td> </tr>
+<tr><td align="center" colspan="3"><code>void</code></td>
```

Comment:
Question: can we do this table in Markdown, or is the HTML necessary?  Same question for all tables in this page.

I like the `<code>` fixes you've done, and removing inline styling.  I'd still like to avoid the inline `center`.

Reply:
Because the walkthrough tables have a complex layout, such as cells that span several columns, they become impossible to realize in the Markdown syntax supported by Docusaurus. You'd need to extend the Markdown syntax to be able to create such tables. I felt #58 could address the design of the table to make them easier to write in Markdown.